plugin.rb

```diff
@@ -213,9 +213,9 @@ def index
 title = GlobalSetting.try(:saml_title) || "SAML"
 button_title = GlobalSetting.try(:saml_button_title) || GlobalSetting.try(:saml_title) || "with SAML"
 
-auth_provider title: button_title,
+auth_provider title: GlobalSetting.try(:saml_title) || button_title,
```

Comment:
`button_title` already contains `saml_title` so I don't see the point of this.

Reply:
`SAML_Title` resolves to "SAML" by default
`button_title` resolves to "with SAML" by default

I used this to set different options in our `app.yml` file.

Reply:
Of note, this PR initially added fallbacks for a number of settings.. many have been incorporated in to master so I've been rebasing this to incorporate those changes.

Agreed that at this point, there's not a huge change/improvement here, but it's still needed in our instance, and i feel can be useful for others as well.